docsbox/docs/views.py

```diff
@@ -54,14 +54,14 @@ def post(self, file_id):
         """
         response = {}
         try:
-            mimetype = request.headers['Content-Type']
-            if mimetype == "application/pdf" or mimetype not in app.config["CONVERTABLE_MIMETYPES"]:
+            mimetype = request.headers.get('Content-Type')
+            if mimetype == None or mimetype == "application/pdf" or mimetype not in app.config["CONVERTABLE_MIMETYPES"]:
                 if request.files and "file" in request.files:
                     mimetype = create_tmp_file_and_get_mimetype(request.files["file"], request.files["file"].filename)['mimetype']
                 elif file_id and is_valid_uuid(file_id):
                     r = get_file_from_via(file_id)
                     if r.status_code == 200:
-                        mimetype = r.headers['Content-Type']
+                        mimetype = r.headers.get('Content-Type')
```

Comment:
Can VIA's Content-type header be undefined?

Reply:
We never had issues here so I assume not, but I decided to change all to get just as a good practice since its not important if we get the content-type from VIA or for example the filename from Content-Disposition so we don't want to trow a error